phpunit/functional/WebhookTest.php

```diff
@@ -297,4 +297,22 @@ public function testWithHLAPIDisabled(): void
         // The main purpose is to test the internal authentication middleware.
         $this->assertNotNull($webhook->getResultForPath('/Administration/User/' . $users_id, 'new', 'User', $users_id));
     }
+
+    public function testGetMonacoSuggestions()
+    {
+        $itemtypes = \Webhook::getItemtypesDropdownValues();
+        $noSuggestions = ['Document_Item'];
```

Comment:
@cconard96 Could you validate that having no suggestions for `Document_Item` is expected?

Reply:
This itemtype may have been broken by https://github.com/glpi-project/glpi/pull/14935 where the "Document_Item" schema was moved to the ManagementController.

Reply:
Do you know where the schema is declared so I can see why document is not returned in ITILController

Reply:
It should be in `ITILController::225` but I don't see any reference to the Document type, maybe this schema is missing.